wide-github.css

```diff
@@ -52,3 +52,13 @@ body:not(.wgh-disabled) .gist-content-wrapper .container-lg {
 body:not(.wgh-disabled) .gist-content-wrapper .container-lg .h-card {
   width: 253px !important;
 }
+
+/* New github design */
+body:not(.wgh-disabled) .container-lg, body:not(.wgh-disabled) .container-xl {
+  margin-left: 0px !important;
```

Comment:
As far as I can tell these `margin-left/right` lines aren't required. Is there a particular page they fix?
I'm trying to keep the CSS minimal, so I'll tweak this in master. Left me know if that breaks something though.